### PR TITLE
feat(Datatable): add dumb reference table

### PR DESCRIPTION
**Pull request by ajkl2533**

### CODE REVIEW

File: src/components/Datatable/Table/temp/IndeterminateCheckbox.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// TODO: move this as next state of the `forms/Checkbox`
+
+import React, { forwardRef, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+
+const useCombinedRefs = (...refs): React.MutableRefObject<HTMLInputElement> => {
```
Comment (PrincAm) [inline src/components/Datatable/Table/temp/IndeterminateCheckbox.tsx:6]: It is a bit tricky part. Please why do we need to combine those references?

**Review by PrincAm — APPROVED** [review]
Great kick-off! Basic functionality is there and working, styles will be updated within next tasks. I just didn't get the handling of IndeterminateCheckbox's references, maybe it would deserve a comment.

File: src/components/Datatable/Table/temp/IndeterminateCheckbox.tsx (hunk 1 above)
Comment (ajkl2533) [reply src/components/Datatable/Table/temp/IndeterminateCheckbox.tsx:6]: Don't worry about that, I'm not sure if we need that at all, it copied from some example

Comment (PrincAm) [reply src/components/Datatable/Table/temp/IndeterminateCheckbox.tsx:5]: Okey then

Comment (ajkl2533) [reply src/components/Datatable/Table/temp/IndeterminateCheckbox.tsx:5]: @PrincAm provided a link with explanation

**Pull request merged**